packages/rum-core/src/boot/preStartRum.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/unbound-method */
```

Comment:
**💬 suggestion:** ‏don't disable ESLint rules for the whole file. You can wrap your code between comments:
```ts
/* eslint-disable ... */
xxx
/* eslint-enable ... */
```